sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/TypeCoercionSuite.scala

```diff
@@ -389,6 +389,25 @@ class TypeCoercionSuite extends AnalysisTest {
     widenTest(StringType, MapType(IntegerType, StringType, true), None)
     widenTest(ArrayType(IntegerType), StructType(Seq()), None)
 
+    widenTest(
+      ArrayType(StringType, containsNull=true),
```

Comment:
please add a whitespace before and after `=` and the same in the following lines

Reply:
Fixed, see dfb4a3b47d88853be348aafd9802f799639693f6